### PR TITLE
Add ZI install

### PR DESCRIPTION
**Pull request by 0xMRTT**
https://wiki.zshell.dev/community/gallery/collection/themes#thp-sindresorhuspure

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -268,6 +268,15 @@ zinit ice compile'(pure|async).zsh' pick'async.zsh' src'pure.zsh'
...
 
+### [zi](https://wiki.zshell.dev)
+
+Update your `.zshrc` file with the following two lines (order matters):
+
+```sh
+zi light-mode for @sindresorhus/pure
+```
+
+See [https://wiki.zshell.dev/community/gallery/collection/themes#thp-sindresorhuspure] for more
```
Comment (sindresorhus) [inline readme.md:279]: Use a titled link

File: readme.md [hunk 2]
```diff
@@ -268,6 +268,15 @@ zinit ice compile'(pure|async).zsh' pick'async.zsh' src'pure.zsh'
 zinit light sindresorhus/pure
 ```
 
+### [zi](https://wiki.zshell.dev)
+
+Update your `.zshrc` file with the following two lines (order matters):
```
Comment (sindresorhus) [inline readme.md:273]: Needs updating

**Review by sindresorhus — CHANGES_REQUESTED** [review]

File: readme.md [hunk 3]
```diff
@@ -268,6 +268,15 @@ zinit ice compile'(pure|async).zsh' pick'async.zsh' src'pure.zsh'
 zinit light sindresorhus/pure
 ```
 
+### [zi](https://wiki.zshell.dev)
```
Comment (mafredri) [inline readme.md:271]: Would you mind removing `zinit` above as well? Zi seems to be the continuation of `zinit` (which is no more).

File: readme.md (hunk 2 above)
Comment (mafredri) [inline readme.md:273]:
```suggestion
Update your `.zshrc` file with the following line:
```

File: readme.md (hunk 3 above)
Comment (0xMRTT) [reply readme.md:271]: I think that we can keep zinit and zi together, even if zi and zinit are forked from zdharma/zinit.

Comment (mafredri) [reply readme.md:271]: Ok, if it's still in use then that's fine. 👍🏻

**Comment by 0xMRTT** [comment]
Do you have any others changes ?

**Pull request merged**